src/core/qgsdataprovider.h

```diff
@@ -492,6 +493,30 @@ class CORE_EXPORT QgsDataProvider : public QObject
      */
     virtual bool renderInPreview( const QgsDataProvider::PreviewContext &context ); // SIP_SKIP
 
+    /**
+     * Retrieve collected Metadata from the Provider source
+     * \brief A structured metadata store for a map layer.
+     * \note
+     * \see QgsVectorLayer::setDataProvider
```

Comment:
Drop these two \see lines - they aren't directly related to this method (they relate to the WHOLE QgsDataProvider class)

Reply:
done